### DISCUSSION TITLE
Unit for reactive and apparent energy

### DISCUSSION THREAD

**Discussion by marko-savic** [grafana/grafana#60833] [Dashboards, Panels & Visualizations]
Hi,
it would be nice to have a unit for reactive energy which is: reactive volt-ampere per hour "VArh" and also kilo version "kVArh."
Same for apparent energy which is: volt-ampere hour 'VAh" and kilo version "kVAh".

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
